pkg/config/config_template.yaml

```diff
@@ -217,7 +218,41 @@ api_key:
 #
 # The timeout to execute the command in second
 # secret_backend_timeout: 5
-
+#
+{{ end -}}
+{{- if .LogsAgent }}
+# Logs agent
+#
+# Enable logs collection, disabled by default
+# logs_enabled: false
+#
+# logs_config:
+#
+## Enable container log collection for all the containers (see ac_exclude to filter out containers)
```

Comment:
I think you didn't pick the right convention :/

I think `##` is used in some places to visually separate more clearly comments from options, but here I think we really want to highlight that the option is indented differently.

This is what I'm looking for typically: https://github.com/DataDog/datadog-agent/blob/f58e99ab6e671167d7b318c9d798b02774772061/pkg/config/config_template.yaml#L576-L591

let me know what you think though

Reply:
@olivielpeau updated with the recommended indentation.